src/js/index.js

```diff
@@ -7,42 +7,20 @@
 import ready from 'lite-ready';
 import { checkForUser } from './userAuth';
 
-
-/**
- * Converts an input to a button in order to improve accessibility.
- *
- * @param {string} selector
- */
-const convertInputToButton = selector => {
-    const input = document.querySelector(selector);
-
-    if (input) {
-        const replaceTag = input.outerHTML.replace(/^<input/, '<button');
-        input.outerHTML = `${replaceTag}</button>`;
-
-        // Query the DOM again for this element now it has changed to a button
-        const button = document.querySelector(selector);
-        button.setAttribute('type', 'button');
-
-        return button;
-    }
-
-    return null;
-};
-
 /**
  * Setup the behaviour for the header component.
  */
 const setupHeader = () => {
-    const menuButton = convertInputToButton('[data-nav-enhance]');
+    const menuButton = document.querySelector('[data-nav-button]');
+    const accessibleButton = document.querySelector('[data-nav-accessible-button]');
+
+    if (accessibleButton) {
+        accessibleButton.classList.remove('is-shown--noJS');
+        accessibleButton.classList.add('is-hidden');
+    }
 
     if (menuButton) {
-        /**
-         * Attach click event handler — as this element is now a button this event will
-         * trigger when the `enter` and `spacebar` keys are pressed.
-         *
-         * @see {@link https://www.w3.org/TR/html51/editing.html#running-synthetic-click-activation-steps - synthetic click activation steps}
-         */
+        menuButton.classList.remove('is-hidden--noJS');
```

Comment:
Do you need to remove this class?  It should only take effect if `noJS` is found right so would have thought you'd be able to leave it as is.

Reply:
gone! 👍